mail.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"mime/multipart"
+	"net/mail"
+	"net/smtp"
+	"net/textproto"
+	"os"
+	"strings"
+	"time"
+
+	b64 "encoding/base64"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	mailSubject      = "MACFin portal test accounts"
+	AttachedFileName = "MACFin_test_accounts.xlsx"
+	bodyEncoding     = "base64"
+	charSet          = "UTF-8"
+)
+
+const (
+	FileMimeType           = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
+	FileContentDisposition = "attachment"
+)
+
+const (
+	SMTPHost = "internal-Enterpris-SMTPProd-I20YLD1GTM6L-357506541.us-east-1.elb.amazonaws.com"
```

Comment:
remove these?

Reply:
yup. Removed since the value comes from the environment variable.